<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GGlaRDDOS</class>
 <widget class="QMainWindow" name="GGlaRDDOS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>GLaRDDOS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>portal.jpg</normaloff>portal.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_Wolf">
              <property name="toolTip">
               <string extracomment="Insanity Wolf"/>
              </property>
              <property name="text">
               <string>Wolf</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_Wolf</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Wolf">
              <property name="toolTip">
               <string extracomment="Targets"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_Wolf">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>1337</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_Wolf">
              <property name="text">
               <string>Random port</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_Greg">
              <property name="toolTip">
               <string extracomment="Good Guy Greg!"/>
              </property>
              <property name="text">
               <string>Greg</string>
              </property>
              <property name="buddy">
               <cstring>lineEdit_Greg</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Greg">
              <property name="toolTip">
               <string comment="Source IPs"/>
              </property>
              <property name="statusTip">
               <string/>
              </property>
              <property name="accessibleName">
               <string/>
              </property>
              <property name="accessibleDescription">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_Greg">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>65535</number>
              </property>
              <property name="value">
               <number>1337</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_Greg">
              <property name="text">
               <string>Random port</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="checkBox_RdPkt">
            <property name="text">
             <string>Random packet</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_fRST">
            <property name="text">
             <string>Force TCP RST</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_maxPkts">
            <property name="text">
             <string>Pkts per combination</string>
            </property>
            <property name="buddy">
             <cstring>spinBox_maxPkts</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_maxPkts">
            <property name="toolTip">
             <string extracomment="Packets to be sent per Wolf &amp; Greg combination"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_Speed">
            <property name="text">
             <string>Speed (pkts/s)</string>
            </property>
            <property name="buddy">
             <cstring>horizontalSlider_Speed</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_Speed">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="pageStep">
             <number>100</number>
            </property>
            <property name="sliderPosition">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval">
             <number>50</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="checkBox_Horde">
            <property name="text">
             <string>Horde</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Portal">
            <property name="text">
             <string>Portal</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_Portal</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Portal">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Room">
            <property name="text">
             <string>Room</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_Room</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Room">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Extension">
            <property name="text">
             <string>Extension</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_Extension</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Extension">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cake">
        <property name="text">
         <string>Bake the cake</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>This cake has been baking since</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="displayFormat">
         <string>dd.MM.yyyy hh:mm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Greetz to Good guy Greg, Insanity Wolf, Mitchunderstood and the guy who combines all three, DSK.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Wolf</tabstop>
  <tabstop>lineEdit_Greg</tabstop>
  <tabstop>spinBox_Wolf</tabstop>
  <tabstop>spinBox_Greg</tabstop>
  <tabstop>checkBox_Wolf</tabstop>
  <tabstop>checkBox_Greg</tabstop>
  <tabstop>checkBox_RdPkt</tabstop>
  <tabstop>checkBox_fRST</tabstop>
  <tabstop>spinBox_maxPkts</tabstop>
  <tabstop>horizontalSlider_Speed</tabstop>
  <tabstop>pushButton_cake</tabstop>
  <tabstop>checkBox_Horde</tabstop>
  <tabstop>lineEdit_Portal</tabstop>
  <tabstop>lineEdit_Room</tabstop>
  <tabstop>lineEdit_Extension</tabstop>
  <tabstop>dateTimeEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
